package.json

```diff
@@ -24,33 +24,33 @@
     ]
   },
   "dependencies": {
-    "@hapi/accept": "^6.0.0",
-    "@hapi/ammo": "^6.0.0",
-    "@hapi/boom": "^10.0.0",
-    "@hapi/bounce": "^3.0.0",
-    "@hapi/call": "^9.0.0",
-    "@hapi/catbox": "^12.0.0",
-    "@hapi/catbox-memory": "^6.0.0",
-    "@hapi/heavy": "^8.0.0",
-    "@hapi/hoek": "^10.0.0",
-    "@hapi/mimos": "^7.0.0",
-    "@hapi/podium": "^5.0.0",
-    "@hapi/shot": "^6.0.0",
-    "@hapi/somever": "^4.1.0",
-    "@hapi/statehood": "^8.0.0",
-    "@hapi/subtext": "^8.0.0",
+    "@hapi/accept": "^6.0.1",
+    "@hapi/ammo": "^6.0.1",
+    "@hapi/boom": "^10.0.1",
+    "@hapi/bounce": "^3.0.1",
+    "@hapi/call": "^9.0.1",
+    "@hapi/catbox": "^12.1.1",
+    "@hapi/catbox-memory": "^6.0.1",
+    "@hapi/heavy": "^8.0.1",
+    "@hapi/hoek": "^11.0.2",
+    "@hapi/mimos": "^7.0.1",
+    "@hapi/podium": "^5.0.1",
+    "@hapi/shot": "^6.0.1",
+    "@hapi/somever": "^4.1.1",
+    "@hapi/statehood": "^8.0.1",
+    "@hapi/subtext": "^8.0.1",
     "@hapi/teamwork": "^6.0.0",
-    "@hapi/topo": "^6.0.0",
-    "@hapi/validate": "^2.0.0"
+    "@hapi/topo": "^6.0.1",
+    "@hapi/validate": "^2.0.1"
   },
   "devDependencies": {
-    "@hapi/code": "^9.0.0",
-    "@hapi/eslint-plugin": "^6.0.0",
-    "@hapi/inert": "^6.0.2",
+    "@hapi/code": "^9.0.3",
+    "@hapi/eslint-plugin": "*",
```

Comment:
Using `*` here caused some problems back when I did the v21 release with a new major version of eslint-plugin— essentially broke CI for the old versions of the modules which weren't compatible with it yet.  I think the idea is that it was supposed to automatically use lab's declared version, not sure why it didn't work!

Reply:
I'm not sure, it worked everywhere I put it 🤷🏻‍♂️